utils/json.go

```diff
@@ -185,6 +192,17 @@ func ExtractObject(old interface{}, path string) interface{} {
 	return nil
 }
 
+// ExtractObjectJMES is used to extract object from old using JMES path
+func ExtractObjectJMES(old interface{}, path string) interface{} {
+	result := make(map[string]interface{}, 1)
+	value, err := jmes.Search(path, old)
+	if err != nil {
+		return nil
+	}
+	result[path] = value
```

Comment:
I have concerns about this part, more details please see: https://github.com/Azure/terraform-provider-azapi/issues/535#issuecomment-2259808879